### PR TITLE
Switch for loop to use unboxed LongIterator

### PR DESCRIPTION
**Pull request by caojohnny**
Found this while browsing through the source code. Should be a rather trivial optimization/change.

### CODE REVIEW

File: Spigot-Server-Patches/0405-Switch-for-loop-to-use-unboxed-LongIterator.patch [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+diff --git a/src/main/java/org/bukkit/craftbukkit/CraftWorld.java b/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
+index 53d4f46d4..3ccb9a545 100644
+--- a/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
++++ b/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
+@@ -16,6 +16,10 @@ import java.util.Set;
+ import java.util.UUID;
+ import java.util.concurrent.CompletableFuture;
+ import java.util.function.Predicate;
++
++import it.unimi.dsi.fastutil.longs.LongIterable;
```
Comment (RealFloorIsJava) [inline Spigot-Server-Patches/0405-Switch-for-loop-to-use-unboxed-LongIterator.patch:16]: missing // Paper comments

maybe use fully qualified names in code?

**Comment by RealFloorIsJava** [comment]
What is the impact of this change? Do you have some benchmarks? :)

**Comment by AgentTroll** [comment]
Completely missed that one oops...

**Comment by caojohnny** [comment]
Discord thinks this is premature optimization

**Pull request closed without merging**